### PR TITLE
aragorn: init at 1.2.38

### PR DESCRIPTION
**Pull request by bzizou**
###### Motivation for this change
Soft needed by the biologists using our computing center (GRICAD) 

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/science/biology/aragorn/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+stdenv.mkDerivation rec {
+  version = "1.2.38";
+  pname = "aragorn";
+
+  src = fetchurl {
+    url = "http://mbio-serv2.mbioekol.lu.se/ARAGORN/Downloads/${pname}${version}.tgz";
+    sha256 = "09i1rg716smlbnixfm7q1ml2mfpaa2fpn3hwjg625ysmfwwy712b";
+  };
+
+  buildInputs = [ gcc ];
```
Comment (aanderse) [inline pkgs/applications/science/biology/aragorn/default.nix]: This isn't required.

**Review by aanderse — CHANGES_REQUESTED** [review]

File: pkgs/applications/science/biology/aragorn/default.nix (hunk 1 above)
Comment (aanderse) [inline pkgs/applications/science/biology/aragorn/default.nix]: Not required.

File: pkgs/applications/science/biology/aragorn/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, gcc }:
```
Comment (aanderse) [inline pkgs/applications/science/biology/aragorn/default.nix]: `gcc` is not required here, already included.

**Review by aanderse — CHANGES_REQUESTED** [review]

**Comment by aanderse** [comment]
@GrahamcOfBorg aragorn

File: pkgs/applications/science/biology/aragorn/default.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{stdenv, fetchurl }:
```
Comment (kalbasit) [inline pkgs/applications/science/biology/aragorn/default.nix]:
```suggestion
{ stdenv, fetchurl }:
```

**Review by kalbasit — CHANGES_REQUESTED** [review]
Just a nitpick. Overall it looks good.

File: pkgs/applications/science/biology/aragorn/default.nix (hunk 3 above)
Comment (bzizou) [reply pkgs/applications/science/biology/aragorn/default.nix]: OK, I commited the suggestion. Should I squash the 2 commits and force-push ?

Comment (aanderse) [reply pkgs/applications/science/biology/aragorn/default.nix]: Please.

**Review by aanderse — APPROVED** [review]

**Pull request merged**